tests/D2L.CodeStyle.Analyzers.Test/Specs/OldAndBrokenServiceLocatorAnalyzer.cs

```diff
@@ -48,29 +48,29 @@ public sealed class BadClass {
 		public BadClass() { }
 
 		public void Uses_OldAndBrokenServiceLocator() {
-			IServiceLocator locator = /* OldAndBrokenLocatorIsObsolete */ OldAndBrokenServiceLocator /**/.Instance;
+			IServiceLocator locator = /* OldAndBrokenLocatorIsObsolete */ OldAndBrokenServiceLocator.Instance /**/;
 		}
 
 		public void Uses_OldAndBrokenServiceLocatorFactory() {
-			IServiceLocator locator = /* OldAndBrokenLocatorIsObsolete */ OldAndBrokenServiceLocatorFactory /**/.Create();
-			Func<IServiceLocator> locatorFactory =  /* OldAndBrokenLocatorIsObsolete */ OldAndBrokenServiceLocatorFactory /**/.Create;
+			IServiceLocator locator = /* OldAndBrokenLocatorIsObsolete */ OldAndBrokenServiceLocatorFactory.Create() /**/;
+			Func<IServiceLocator> locatorFactory =  /* OldAndBrokenLocatorIsObsolete */ OldAndBrokenServiceLocatorFactory.Create /**/;
 		}
 
 		public void Uses_IObjectActivator() {
-			/* OldAndBrokenLocatorIsObsolete */ IObjectActivator /**/ activator = default;

```

Comment:
Referencing the types `IObjectActivator` and `ICustomObjectActivator` will no longer causing issues.  Just where they are used.